graph_def_editor/rewrite.py

```diff
@@ -570,7 +572,8 @@ def fold_batch_norms_up(g):
   ```
     ReLU => Min(6 / multiplier from batch norm)
   """
-  def compute_input_dim(n: node.Node):
+  def compute_input_dim(n #type: node.Node
+                        ):
     if n.op_type == "Conv2D" or n.op_type == "DepthwiseConv2dNative":
```

Comment:
Since there's only one argument here, I'd put the type annotation on the next line, i.e.
```python
def compute_input_dim(n):
    # type: (node.Node) -> int
```